templates/frontend/components/registrationForm.tpl

```diff
@@ -16,112 +16,112 @@
+            <label>
+                <span class="label">
+                    {translate key="common.country"}
+                    <span class="required" aria-hidden="true">*</span>
+                    <span class="pkp_screen_reader">
+                        {translate key="common.required"}
+                    </span>
+                </span>
+                <select name="country" id="country" required aria-required="true">
+                    <option></option>
+                    {html_options options=$countries selected=$country}
+                </select>
+            </label>
+        </div>
+    </div>
 </fieldset>
 
 <fieldset class="login">
-	<legend>
-		{translate key="user.login"}
-	</legend>
-	<div class="fields">
-		<div class="email">
-			<label>
-				<span class="label">
-					{translate key="user.email"}
-					<span class="required" aria-hidden="true">*</span>
-					<span class="pkp_screen_reader">
-						{translate key="common.required"}
-					</span>
-				</span>
-				<input type="email" name="email" id="email" value="{$email|escape}" maxlength="90" required aria-required="true">
-			</label>
-		</div>
-		<div class="username">
-			<label>
-				<span class="label">
-					{translate key="user.username"}
-					<span class="required" aria-hidden="true">*</span>
-					<span class="pkp_screen_reader">
-						{translate key="common.required"}
-					</span>
-				</span>
-				<input type="text" name="username" id="username" value="{$username|escape}" maxlength="32" required aria-required="true">
-			</label>
-		</div>
-		<div class="password">
-			<label>
-				<span class="label">
-					{translate key="user.password"}
-					<span class="required" aria-hidden="true">*</span>
-					<span class="pkp_screen_reader">
-						{translate key="common.required"}
-					</span>
-				</span>
-				<input type="password" name="password" id="password" password="true" maxlength="32" required aria-required="true">
-			</label>
-		</div>
-		<div class="password">
-			<label>
-				<span class="label">
-					{translate key="user.repeatPassword"}
-					<span class="required" aria-hidden="true">*</span>
-					<span class="pkp_screen_reader">
-						{translate key="common.required"}
-					</span>
-				</span>
-				<input type="password" name="password2" id="password2" password="true" maxlength="32" required aria-required="true">
-			</label>
-		</div>
-	</div>
+    <legend>
+        {translate key="user.login"}
+    </legend>
+    <div class="fields">
+        <div class="email">
+            <label>
+                <span class="label">
+                    {translate key="user.email"}
+                    <span class="required" aria-hidden="true">*</span>
+                    <span class="pkp_screen_reader">
+                        {translate key="common.required"}
+                    </span>
+                </span>
+                <input type="email" name="email" id="email" value="{$email|escape}" maxlength="90" required aria-required="true">
```

Comment:
Looks like there's also an autocomplete option for email that should be used here: https://www.w3.org/TR/WCAG21/#input-purposes